ViewBase.js

```diff
@@ -41,23 +41,23 @@ define(["require", "dojo/when", "dcl/dcl", "dojo/Deferred", "dapp/utils/view"],
 				if (this._started) {
 					return this;
 				}
-				this._startDef = new Deferred();
-				when(this.load(), function (controller) {
-					this._createDataStores();
-					this._startup(controller);
+				return new Promise(function (resolve) {
+					this._startResolve = resolve;
+					return this.load().then(function (controller) {
+						this._createDataStores();
+						this._startup(controller);
+					}.bind(this));
 				}.bind(this));
```

Comment:
can be simplified to:


				this._startResolve = resolve;
				return this.load().then(function (controller) {
					this._createDataStores();
					this._startup(controller);
				}.bind(this));

in other words remove lines 44 and 50